rules/windows/persistence_startup_folder_file_written_by_unsigned_process.toml

```
@@ -14,6 +14,49 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Startup Folder Persistence via Unsigned Process"
+note = """## Triage and analysis
+
+### Investigating Startup Folder Persistence via Unsigned Process
+
+Windows Startup folder is a special folder in Windows. Programs added to this folder are executed during the logon of an
+account, without user interaction, making it an excellent opportunity for attackers to maintain persistence.
+
+This rule will monitor for unsigned processes writing to the startup folder locations.
+
+#### Possible investigation steps:
+
+- Investigate process execution chain (parent process tree).
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Validate the activity is not related to planned patches, updates, network administrator activity or legitimate software
+installations.
+- Determine if activity is unique by validating if other machines in same organization have similar entry.
+- Retrieve the file:
+  - Use a sandboxed malware analysis system to perform analysis.
+    - Observe attempts of contacting external domains and addresses.
+  - Use the PowerShell Get-FileHash cmdlet to get the SHA-256 hash value of the file.
+    - Search for the existence of this file in resources like VirusTotal, Hybrid-Analysis, CISCO Talos, Any.run, etc.
+
+### False Positive Analysis
+
+- There is a high possibility of benign legitimate programs being added to Startup folders. This activity could be based
+on new software installations, patches, or any kind of network administrator related activity. Before entering further
+investigation, it should be verified that this activity is not benign.
+
+### Related Rules
+
+- Suspicious Startup Shell Folder Modification
+- Persistent Scripts in the Startup Directory
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- If the triage identified malware, search the environment for additional compromised hosts.
+  - Implement any temporary network rules, procedures, and segmentation required to contain the malware.
+  - Immediately block the IoCs identified.
```

```suggestion
  - Immediately block external network domains and addresses communicating with malware if optional.
```

---

Depending on [the discussion above](https://github.com/elastic/detection-rules/pull/1871/files#r833637623), if we're staying with "IoCs" I suggest spelling out the term first:
```suggestion
  - Immediately block the identified indicators of compromise (IoCs).
```